src/Benchmarks/Startup.cs

```diff
@@ -108,6 +109,78 @@ public void ConfigureServices(IServiceCollection services)
 
                     break;
 #endif
+
+                case DatabaseServer.Sqlite:
```

Comment:
Should I add a check for `Cache=Shared` in the connection string?

Reply:
We know what we are doing. And we also might want to test that optionally. So no.